samza-core/src/main/java/org/apache/samza/container/grouper/task/BalancingTaskNameGrouper.java

```diff
@@ -54,5 +55,11 @@
    * @param localityManager provides a persisted task to container map to use as a baseline
    * @return                the grouped tasks in the form of ContainerModels
    */
-  Set<ContainerModel> balance(Set<TaskModel> tasks, LocalityManager localityManager);
+  default Set<ContainerModel> balance(Set<TaskModel> tasks, LocalityManager localityManager) {
```

Comment:
right, I was cleaning this file on Navina's suggestion, but didn't clean completely. Fixing.